pkg/curatedpackages/packageinstaller.go

```diff
@@ -47,9 +56,27 @@ func newPackageController(installer ChartInstaller, runner KubectlRunner, spec *
 	return NewPackageControllerClient(installer, runner, kubeConfig, imageUrl, chart.Name, chart.Tag())
 }
 
+func newCertManager(runner KubectlRunner, spec *cluster.Spec) *CertManagerClient {
+	kubeConfig := kubeconfig.FromClusterName(spec.Cluster.Name)
+
+	return NewCertManagerClient(runner, kubeConfig)
+}
+
 func (pi *Installer) InstallCuratedPackages(ctx context.Context) error {
 	PrintLicense()
-	err := pi.installPackagesController(ctx)
+
+	// If cert-manager does not exist, instruct users to follow instructions in
+	// PrintCertManagerDoesNotExistMsg to install packages manually.
+	certManagerExits, err := pi.certManager.CertManagerExists(ctx)
+	if err != nil {
+		return err
```

Comment:
Can you add one test in case cert manager doesn't exist?

Reply:
Does `TestPackageInstallerFailWhenCertManagerFails` in `packageinstaller_test.go` suffice this request or would you like me to add additional test?

Reply:
Well it does seem like line 64 is not covered in codecoverage so it might not be sufficient